src/asmcup/runtime/Main.java

```diff
@@ -7,7 +7,6 @@ public static void main(String[] args) throws IOException {
 		if (args.length != 4) {
 			System.err.printf("USAGE: asmcup-runtime <scenario>\n");
 			System.exit(1);
-			return;

```

Comment:
`return` is usually left so that Java and other static analysis tools know execution would end.